Makefile

```diff
@@ -78,7 +85,7 @@ rpm: clean dist ##             - Creates a local RPM package, useful for develop
 ostree: qm_dropin_img_tempdir ##             - A helper for creating QM packages for ostree based distros
 
 .PHONY: qm_dropin_window_manager
-qm_dropin_window_manager: qm_dropin_mount_bind_kvm qm_dropin_mount_bind_sound qm_dropin_mount_bind_tty7 qm_dropin_mount_bind_input ##         - QM RPM sub-package qm_dropin_window_manager
+qm_dropin_window_manager: qm_dropin_mount_bind_kvm qm_dropin_mount_bind_sound qm_dropin_mount_bind_tty7 qm_dropin_mount_bind_input qm_dropin_mount_bind_video0 ##         - QM RPM sub-package qm_dropin_window_manager
 	sed -i 's/%define enable_qm_window_manager 0/%define enable_qm_window_manager 1/' ${SPECFILE}
```

Comment:
I think that you may add an @ symbol before the sed and $(MAKE) commands to suppress the command output and only show the command if it fails.
`@sed -i 's/%define enable_qm_window_manager 0/%define enable_qm_window_manager 1/' ${SPECFILE}`

Reply:
I proposed different solution 
in here https://github.com/containers/qm/pull/612
better to use macros
Packit, doe not use the makefile to build rpms